org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/hover/FocusableBrowserInformationControl.java

```diff
@@ -94,57 +94,79 @@ protected void createContent(Composite parent) {
 	}
 
 	private void updateBrowserSize(final Browser browser) {
+		updateBrowserSize(browser, 1);
+	}
+
+	private void updateBrowserSize(final Browser browser, final int attempt) {
```

Comment:
@sebthom , can you rebase your PR? Now that I think I have solved the problem with `safeEvaluate` returning exceptions, I think you can go back to your original attempt which was much simpler.